spec/components/pretty_text_spec.rb

```diff
@@ -1290,17 +1296,25 @@ def strip_image_wrapping(html)
         </li>
         </ul>
         <p><img src="#{upload.url}" alt="upload"></p>
```

Comment:
I think it is worth explicitly configuring a CDN here so it is clear that the links that come back are not CDN-ified.

Reply:
It is impossible for `Upload#url` to return a CDN-ified URL here.

Reply:
Sorry I mean set the global setting so the site has a CDN this confirms that when cooking you never cdnify things.

Reply:
OK done :+1: